src/components/newSidebar/manageFile/ManageFile.jsx

```diff
@@ -214,6 +214,13 @@ const ManageFile = () => {
     );
   }, [itemsCountToDisplay]);
 
+  console.log("allDocumentsArray", allDocumentsArray)
+
+
+  console.log("allWorkflowsArray", allWorkflowsArray)
+  console.log("allTemplatesArray", allTemplatesArray)
+  console.log("allProcesses", allProcesses)
+  
```

Comment:
Unnecessary logs should not go to production

Reply:
resolved